Code/wwlib/mpu.cpp

```diff
@@ -89,7 +89,11 @@ unsigned long Get_CPU_Clock(unsigned long & high)
 {
 	int h;
 	int l;
-	_int64 tsc = __rdtsc();
+#ifdef _MSC_VER
```

Comment:
I'd be inclined to move this ifdef magic to a compat header similar to how its done here https://github.com/TheSuperHackers/GeneralsGameCode/blob/487e5ce2daba29ed8f196252580d5d0f0f4a4dc9/Dependencies/Utility/Utility/intrin_compat.h#L93

That will also allow implementing this for other CPU arcitectures without further code modification.

Reply:
I think `intrin_compat.h` is the wrong approach as we want to get rid of the Intel x86 addiction.

I did not put much thought into this as we'll probably introduce a `Platform` namespace where we can implement getticks-platform-dependent implementations.
e.g. `SDL_GetPerformanceCounter()` for SDL, and the `__rdtsc` thing for win32 x86 (I don't know about a arm64 equivalent intrinsic)

Reply:
I think I addressed this nicely: I replaced all usages of `__rdtsc` with win32's `QueryPerformanceCounter` function.

The only usage of `intrin.h` is in `cpudetect.c` where it used for cpu speed measurement (unreliable these days with reclocking) and cpuid.  It's only used for Intel architecture, so arm ready.

Reply:
This PR is ready, please review.

> ... I'd prefer we track with what SuperHackers have done to solve some of these issues already to keep the code bases similar where they share code. Calling platform::ticks or __rdtsc whatever its called doesn't really matter in the grand scheme of things.

True. There's a small difference between this project and Generals: we don't support MSVC6 anymore (luckily).